common/app/model/meta/LinkedData.scala

```diff
@@ -62,13 +78,80 @@ object PotentialAction {
 }
 
 case class ItemList(
+  `@context`: String = "http://schema.org",
+  `@type`: String = "ItemList",
   url: String,
   itemListElement: Seq[ListItem]
-) extends LinkedData("ItemList")
+) extends LinkedData
+
+object ItemList {
+  implicit val formats: OFormat[ItemList] = Json.format[ItemList]
+}
 
 case class ListItem(
   `@type`: String = "ListItem",
   position: Int,
   url: Option[String] = None,// either/or url and item, but needs some care serialising
   item: Option[ItemList] = None
 )
+
+object ListItem {
+  implicit val formats: OFormat[ListItem] = Json.format[ListItem]
+}
+
+case class Person(
+  `@type`: String = "Person",
+  name: String,
+  sameAs: String,
+)
+
+object Person {
+  implicit val formats: OFormat[Person] = Json.format[Person]
+}
+
+case class IsPartOf(
+  `@type`: List[String] = List("CreativeWork", "Product"),
+  name: String = "The Guardian",
+  productID: String = "theguardian.com:basic"
```

Comment:
Maybe to review in later schema work but I don't get product makes sense here...

Reply:
Yeah I don't really know much about this bit to be honest.